src/ZoneServer/Buffs/Handlers/OverReinforce_Buff.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Melia.Shared.Tos.Const;
+using Melia.Zone.Buffs.Base;
+using Melia.Zone.Scripting;
+using Melia.Zone.World.Actors.Characters;
+
+namespace Melia.Zone.Buffs.Handlers
+{
+	/// <summary>
+	/// Handle for the Over Reinforce Buff, which increases the target's physical and magic attack
+	/// </summary>
+	[BuffHandler(BuffId.OverReinforce_Buff)]
+	public class OverReinforce_Buff : BuffHandler
+	{
+		private float attackBonus = 0;
+
+		public override void OnStart(Buff buff)
+		{
+			var target = buff.Target as Character;
+
+			if (target != null)
+			{
+				// Based on in game values (tooltip)
+				attackBonus = 1308 + (349 * buff.Data.Level);
```

Comment:
I'm starting to wonder whether we're using different games for referencing bonuses, because Over Reinforce is said to raise ATK by X on my client. I can't tell where the 1308 is coming from 🤔 That seems way, way too high. I'll stop the review here for the moment because there are clearly several issues to work out still with this PR.

![Screenshot 2023-07-03 132349](https://github.com/NoCode-NoLife/melia/assets/5593920/7af6f747-fd3c-470d-9f0f-6819e0ad12e2)

Reply:
![image](https://github.com/NoCode-NoLife/melia/assets/29080604/83e17318-e769-4342-b758-7a84c27324af)
 Good point, I need to figurate out this formula.

Reply:
I will follow the calc_property_skill.lua:

```
function SCR_GET_OverReinforce_Ratio(skill)
    local pc = GetSkillOwner(skill)
    local maxpatk = SCR_Get_DEFAULT_MAXPATK(pc, 0)
    local minpatk = SCR_Get_DEFAULT_MINPATK(pc, 0)
    local rate = 0.015 + skill.Level * 0.004

    local vibora_rate = GetExProp(pc, 'ITEM_VIBORA_Empowering') / 1000
    if vibora_rate > 0 then
        rate = rate + vibora_rate
    end

    local value = ((maxpatk + minpatk)/2) * rate
    
    return math.floor(value);
end
```

Reply:
Can you tell me which version of the game that is? Because as you can see, my client shows _vastly_ different values, both for the bonus as well as for the SP usage.

Reply:
I think its showing different values because I am max level with that character.

Reply:
Ah, right. I overlooked that this value is dependent on your own PATK.